GoogleChromeLabs/ps-analysis-tool
# Add topic list

## Description

Add topics list section to show list of topics using the [Topics API ](https://developer.chrome.com/en/docs/privacy-sandbox/topics/)

## Relevant Technical Choices



## Testing Instructions

To test, visit https://topics-demo.glitch.me/ and follow the instructions to enable topics and visit the given URLs.
Then go to chrome://topics-internals/ and click on "Calculate Now".
The topics will only appear in the PS panel if you are able to make it work for https://topics-demo.glitch.me/

## Additional Information:



## Screenshot/Screencast

<img width="1084" alt="image" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/6297436/b23a231f-e203-47a7-8e2a-c9a30296aee6">




Fixes #100
